src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/EditContactDetailsModal/EditContactDetailsModal.tsx

```diff
@@ -55,72 +58,141 @@ const PrimaryContactIcon = styled(BookmarkIcon)(({ theme }) => ({
   color: theme.palette.cruGrayMedium.main,
 }));
 
+const LoadingIndicator = styled(CircularProgress)(({ theme }) => ({
+  margin: theme.spacing(0, 1, 0, 0),
+}));
+
 interface EditContactDetailsModalProps {
   contact: ContactDetailsTabQuery['contact'];
+  accountListId: string;
   isOpen: boolean;
-  handleOpenModal: (open: boolean) => void;
+  handleClose: () => void;
 }
+
 export const EditContactDetailsModal: React.FC<EditContactDetailsModalProps> = ({
+  accountListId,
   contact,
   isOpen,
-  handleOpenModal,
+  handleClose,
 }): ReactElement<EditContactDetailsModalProps> => {
   const { t } = useTranslation();
+  const { enqueueSnackbar } = useSnackbar();
+  const [
+    updateContact,
+    { loading: updating },
+  ] = useUpdateContactDetailsMutation();
+
+  const contactSchema: yup.SchemaOf<
+    Pick<ContactUpdateInput, 'name' | 'id'>
+  > = yup.object({
+    name: yup.string().required(),
+    id: yup.string().required(),
+  });
+
+  // TODO Add logic to update primary person when it comes available in mutation
```

Comment:
What's the status of this? Is it a ticket?

Reply:
Yeah, I created https://jira.cru.org/browse/MPDX-7071 and Spencer may be able to get to it later this week.

Reply:
👍 Feel free to stick a comment here with the Jira link. I think it's clearer when someone stumbles accross it later.